apps/src/applab/applab.js

```diff
@@ -1879,6 +1888,27 @@ Applab.onDesignModeClear = function() {
   document.getElementById('divApplab').innerHTML = Applab.levelHtml = "";
 };
 
+Applab.onDesignModeManageAssets = function(assetChosen, typeFilter) {
+  var codeDiv = document.createElement('div');
+  var showChoseImageButton = assetChosen && typeof assetChosen === 'function';
+  var dialog = studioApp.createModalDialog({
+    Dialog: Dialog,
+    contentDiv: codeDiv,
+    icon: studioApp.icon,
```

Comment:
Could probably also have studio.createModalDialog default to using studioApp.icon